buildbot/build_tflite.sh

```diff
@@ -97,6 +97,7 @@ ninja -C eigen/src/eigen-build install
 # ARM_NEON_2_x86_SSE
 git clone --filter=tree:0 --no-checkout ${NEON_2_SSE_REPOSITORY} ARM_NEON_2_x86_SSE/src/ARM_NEON_2_x86_SSE
 git -C ARM_NEON_2_x86_SSE/src/ARM_NEON_2_x86_SSE checkout ${NEON_2_SSE_TAG}
+sed -i -e 's/3.0/3.5/' ARM_NEON_2_x86_SSE/src/ARM_NEON_2_x86_SSE/CMakeLists.txt
```

Comment:
Can you leave a `TODO` comment above this line pointing to https://github.com/intel/ARM_NEON_2_x86_SSE/pull/80?

Reply:
Good point. Done.